### PR TITLE
Merge master changes into 5.0-dev

### LINKED ISSUE #803: Merge the latest master changes to 5.0-dev
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 5.0-dev | Manager/API/Agent | Install/Upgrade/Remove | All |

There is a problem when executing Jenkins checks when a PR points to `5.0-dev`. The following error appears:

```
    Installed (but unpackaged) file(s) found:
   /var/ossec/tmp/sca-5.0.0-0.commitf941835-tmp/sles/15/cis_sles15_linux.yml
```

This is affecting the following checks:

- Build Linux x86_64 agent rpm package
- Build Linux x86_64 manager rpm package

It seems to be a problem related to missed changes present in `master` branch but not in `5.0-dev` branch.

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|Closes #803 |

## Description

This PR merges the changes from branches master and 4.2 into5.0-dev branch

## Tests

- Build the package in any supported platform
  - [x] Linux
  - [x] Windows
  - [x] macOS
  - [x] Solaris

### CODE REVIEW

**Review by okynos — CHANGES_REQUESTED** [review]
GJ, review comments.

File: rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec [hunk 1]
```diff
@@ -105,16 +105,13 @@ mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/packages_files/manager_installation_
...
-mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp/debian/{7,8,9}
-mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp/ubuntu/{12,14,16}/04
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp/rhel/{8,7,6,5}
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp/sles/{11,12}
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp/suse/{11,12}
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp/sunos
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp/windows
 
-cp -r ruleset/sca/{applications,generic,centos,darwin,debian,rhel,sles,sunos,windows} ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp
+cp -r ruleset/sca/{applications,generic,mongodb,nginx,oracledb,centos,darwin,debian,rhel,sles,sunos,windows} ${RPM_BUILD_ROOT}%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec:114]: Yes, the manager includes all of the SCA files.

File: debs/SPECS/5.0.0/wazuh-manager/debian/changelog [hunk 2]
```diff
@@ -82,6 +82,12 @@ wazuh-manager (4.0.0-RELEASE) stable; urgency=low
 
  -- Wazuh, Inc <info@wazuh.com>  Mon, 19 Oct 2020 06:59:39 +0000
 
+wazuh-manager (3.13.3-RELEASE) stable; urgency=low
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply debs/SPECS/5.0.0/wazuh-manager/debian/changelog:85]: No, that error only appears in RPM packages.

File: rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec [hunk 3]
```diff
@@ -560,7 +552,9 @@ fi
 rm -fr %{buildroot}
 
 %files
+%defattr(-,root,wazuh)
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec:555]: Done in 6bffa74

File: debs/SPECS/5.0.0/wazuh-manager/debian/changelog [hunk 4]
```diff
@@ -18,33 +18,33 @@ wazuh-manager (4.2.0-RELEASE) stable; urgency=low
...
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
+-- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
 
 wazuh-manager (4.1.1-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply debs/SPECS/5.0.0/wazuh-manager/debian/changelog:45]: Done in 6bffa74

File: rpms/SPECS/5.0.0/wazuh-agent-5.0.0.spec [hunk 5]
```diff
@@ -475,11 +469,17 @@ if [ -d %{_localstatedir}/queue/ossec ]; then
...
 
+if [ -f %{_sysconfdir}/ossec-init.conf ]; then
+  rm -rf %{_sysconfdir}/ossec-init.conf
+fi
+
 %clean
 rm -fr %{buildroot}
 
 %files
+%defattr(-,root,root)
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply rpms/SPECS/5.0.0/wazuh-agent-5.0.0.spec:480]: Done in 6bffa74

File: debs/SPECS/5.0.0/wazuh-agent/debian/changelog [hunk 6]
```diff
@@ -1,50 +1,50 @@
...
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
 
- -- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
+-- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
 
 wazuh-agent (4.1.1-RELEASE) stable; urgency=low
 
-  * More info: https://documentation.wazuh.com/current/release-notes/
+ * More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (okynos) [inline debs/SPECS/5.0.0/wazuh-agent/debian/changelog:45]: Fix tabulations please
image

File: debs/SPECS/5.0.0/wazuh-manager/debian/changelog (hunk 4 above)
[the remark this answers is not in the document]
Comment (okynos) [reply debs/SPECS/5.0.0/wazuh-manager/debian/changelog:45]: Same here, review please.

File: rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec (hunk 1 above)
[the remark this answers is not in the document]
Comment (okynos) [reply rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec:114]: Do you have to add `amzn` here?

**Review by okynos — CHANGES_REQUESTED** [review]
Minor changes, GJ.

File: debs/SPECS/5.0.0/wazuh-agent/debian/changelog (hunk 6 above)
Comment (DFolchA) [reply debs/SPECS/5.0.0/wazuh-agent/debian/changelog]: Done in b335d2b

File: rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec (hunk 1 above)
[the remark this answers is not in the document]
Comment (DFolchA) [reply rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec]: Done in b335d2b

**Review by okynos — APPROVED** [review]
LGTM.

**Review by okynos — APPROVED** [review]
LGTM.

**Pull request merged**